docs/user-guide/query-data/promql.md

```diff
@@ -154,6 +154,8 @@ Both instant and range selector are supported. But notice that in both Prometheu
 
 Time duration and offset are supported, but `@` modifier is not supported yet.
 
+When selecting non-existent columns, they will be treated as columns filled with empty string values (`""`).
```

Comment:
```suggestion
When selecting non-existent columns, they will be treated as columns filled with empty string values (`""`). This behavior aligns with both Prometheus and VictoriaMetrics.
```

Reply:
resolved in [ea424e1](https://github.com/GreptimeTeam/docs/pull/1684/commits/ea424e1d7a1d703a0af9d17383746f47eb1a5a95)